spec/Functions.tex

```diff
@@ -786,45 +786,26 @@ \subsection{The Const Ref Return Intent}
 form of the \chpl{ref} return intent. Calls to functions marked with
 the \chpl{const ref} return intent are not lvalue expressions.
 
-\subsection{Choosing a Ref Return Intent Function Based on Calling Context}
-\label{Ref_Pair}
-\index{functions!ref-pair}
-
-In some situations, it is useful to distinguish between calls that are
-used in an lvalue setting and those that are not. In particular, suppose
-that there are two functions that have the same formal arguments and
-differ only in their return intent. Normally, such a situation will cause
-an error since it is ambiguous which function is called. However, the
-Chapel language includes a special rule for determining which function to
-call when one of the calls is an lvalue and the other is not.  This rule
-enables data structures such as sparse arrays.
-
-This special rule is called the ref-pair rule. If, when resolving a call,
-there are two candidate functions where:
-
-\begin{itemize}
-\item
- one of them uses the \chpl{ref} return intent and the other uses
- \chpl{const}, \chpl{const ref}, or blank return intent
-\item
- either both use promotion or neither do
-\end{itemize}
-
-then this pair of candidate functions is called a ref-pair.
-
-When such a call appears on the left-hand side of an assignment
-statement, is returned in a function with \chpl{ref} return intent, or
-appears in a call and corresponds to a formal argument with \chpl{out},
-\chpl{inout}, or \chpl{ref} intent, the candidate with the \chpl{ref}
-return intent is used. Otherwise, the other candidate is used.
-
-% TODO - this section, or a significant portion of it,
-% should move to the function resolution section.
+\subsection{Return Intent Overloads}
+\label{Return_Intent_Overloads}
+\index{functions!return intent overloads}
+
+In some situations, it is useful to choose the function called based upon
+how the returned value is used.  In particular, suppose that there are
+two functions that have the same formal arguments and differ only in
+their return intent. Normally, such a situation will cause an error since
```

Comment:
It seems like we could say something better than "Normally" here. Maybe something like "In other languages, " ?